src/runtime_src/core/include/unittests/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022, Advanced Micro Devices, Inc. All rights reserved.
+
+if (NOT WIN32)
+  add_subdirectory(xrt_h_c_compilation)
```

Comment:
I suggest you rename `xrt_h_c_compilation` to `xrt_c_compilation`.  I have several other header files I would like to add to this test.